examples/housing_market_research/investor_analytics/portfolio_analysis.ipynb

```diff
@@ -0,0 +1,864 @@
+    "\n",
+    "# Update trace properties\n",
+    "fig.update_traces(marker=dict(line=dict(width=0)))\n",
+    "\n",
+    "# Customize the layout of the chart\n",
+    "fig.update_layout(\n",
+    "    margin=dict(l=40, r=40, t=100, b=40),  # Adjust margins to fit the subtitle\n",
+    "    showlegend=False,\n",
+    "    title={\n",
+    "        'text': 'Rate of Single-Family Housing Stock Growth Since 2010',  # Corrected title text\n",
+    "        'y': 0.95,\n",
+    "        'x': 0.5,\n",
+    "        'xanchor': 'center',\n",
+    "        'yanchor': 'top',\n",
+    "        'font': style_config['title_font']\n",
+    "    },\n",
+    "    xaxis=dict(\n",
+    "        title_text='',\n",
+    "        tickformat='.2%',\n",
+    "        showgrid=style_config['showgrid'],\n",
+    "        gridwidth=style_config['gridwidth'],\n",
+    "        gridcolor=style_config['grid_color'],\n",
+    "        linecolor=style_config['line_color_axis'],\n",
+    "        linewidth=style_config['linewidth'],\n",
+    "        titlefont=style_config['title_font_axis'],\n",
+    "        tickfont=dict(size=style_config['axis_font']['size'], color=style_config['axis_font']['color']),\n",
+    "    ),\n",
+    "    yaxis=dict(\n",
+    "        title_text='% Housing Stock Growth',\n",
+    "        showgrid=style_config['showgrid'],\n",
+    "        gridwidth=style_config['gridwidth'],\n",
+    "        gridcolor=style_config['grid_color'],\n",
+    "        tickfont=style_config['axis_font'],\n",
+    "        zeroline=False,\n",
+    "        tickformat='.1%',\n",
+    "        linecolor=style_config['line_color_axis'],\n",
+    "        linewidth=style_config['linewidth'],\n",
+    "        titlefont=style_config['title_font_axis']\n",
+    "    ),\n",
+    "    plot_bgcolor=style_config['background_color'],\n",
+    "    paper_bgcolor=style_config['background_color'],\n",
+    "    font=dict(color=style_config['font_color']),\n",
+    "    legend_title_text='',\n",
+    "    autosize=False,\n",
+    "    width=1600,\n",
+    "    height=800,\n",
+    "    title_font=dict(size=24),\n",
+    "    xaxis_title_font=dict(size=18),\n",
+    "    yaxis_title_font=dict(size=18),\n",
+    "    legend_title_font=dict(size=14),\n",
+    "    legend_font=dict(size=12),\n",
+    "    legend=dict(\n",
+    "        x=style_config['legend_x'],\n",
+    "        y=style_config['legend_y'],\n",
+    "        xanchor=style_config['legend_xanchor'],\n",
+    "        yanchor=style_config['legend_yanchor'],\n",
+    "        font=style_config['legend_font'],\n",
+    "        bgcolor='rgba(0, 0, 0, 0)'\n",
+    "    ),\n",
+    ")\n",
+    "\n",
+    "# Set the background color to black\n",
+    "fig.add_layout_image(create_labs_logo_dict())\n",
+    "\n",
+    "# Display the chart\n",
+    "fig.show()\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, you can create a horizontal bar chart using `matplotlib`."
```

Comment:
I think you can remove this cell entirely, which will drop the PIL dependency and help keep it scoped to creating one type of chart. If you want to keep horizontal as an option, I would adjust the original plotly code to do the same as opposed to using multiple charting libraries. Plotly is a required dependency in the SDK install, so folks should have it locally.

Reply:
removed